mlir/lib/Conversion/GPUToSPIRV/GPUToSPIRVPass.cpp

```diff
@@ -54,22 +55,63 @@ void GPUToSPIRVPass::runOnOperation() {
 
   SmallVector<Operation *, 1> gpuModules;
   OpBuilder builder(context);
+
+  auto getTargetEnvFromGPUModuleOp = [=](gpu::GPUModuleOp moduleOp) {
```

Comment:
Can you list the captured variables explicitly?

Reply:
Done.